### PR TITLE
Add very basic docs

### PR DESCRIPTION
**Pull request by jgsogo**
Add docs for #100 
related #99 

This PR adds very basic docs in a `docs` folder at the root of the repository. Also modifies the `README.md` file pointing to it and removing some information that could be outdated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,25 +51,20 @@ It will output the `.vsix` file to:
...
-Update 03-14-2019
-------------------
-Thanks to @sboulema and @SSE4, the extension now provides the minimum required functionality for basic use and initial testing. 
-
-Update 02-06-2019
-------------------
-Plugin development is now being resumed after being stagnant for the past year. 
-
-The overall goal of the extension is for Visual Studio to be able to execute Conan automatically as-needed based on the currently loaded solution/project/configuration.  Over time, this could grow to a lot of convenience operations.  However, the primary (first) objective is to run `conan install` which will generate `conanbuildinfo.props` and satisfy the dependencies. 
+This example has a `conanfile.txt` with a dependency on the popular formatting library `fmt`. When the project is first opened, Visual Studio is unable to find the `fmt` project dependency. The extension should immediately run however, and Visual Studio should give you a toolbar dropdown message offering to refresh the project. After refreshing, the `fmt` headers should be found by the compiler, and the `lib` files should be found by the linker successfully.
```
Comment (SSE4) [inline README.md]: this is no longer the true, there is no toolbsar dropdown and there is no need to reload the project

File: README.md [hunk 2]
```diff
@@ -42,25 +51,20 @@ It will output the `.vsix` file to:
...
 
-Thus, the first requirement is to provide users a mechanism for mapping each Solution/Project/Configuration to a corresponding `conan install` command. A strategy has been chosen for this, and is being discussed here: https://github.com/conan-io/conan-vs-extension/issues/5
+Changelog
+---------
 
-Overall, future features and should try to use a similar configuration-file-based strategy to provide maximum configurability and flexibility to the user of the extension, by making any new conan-related-operations exposed in any toolbar menus and right-click menus configurable and composeable.  This is particular important in the near-term while we are still deciding how the parts should work together, so that new workflow ideas can be tested without requiring code changes and rebuilds of the extension. 
+This extension started was started by community members like @sboullema, @SSE4, @ForNeVer and @solvingj, thanks to
```
Comment (SSE4) [inline README.md]: I don't think it's useful information for the changelog, we may just omit it or move somewhere else (below or to contributors file)

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+# Conan Extension for Visual Studio
+
+Conan Extension for Visual Studio provides a transparent integration between a Visual Studio
+project and the [Conan C/C++ package manager](https://docs.conan.io/en/latest/index.html). It
+automates the retrieval of Conan dependencies matching the configuration of the Visual Studio
+project and add the required include directories, linker ones and libraries.
```
Comment (SSE4) [inline docs/README.md]:
```suggestion
project and adds the required include directories, linker ones and libraries.
```

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      "Debug|x64": "debug_x64",
+      "Release|Win64": "C:/conan/profiles/release_win64"
+   }
+}
+```
+
+Under the `"configurations"` key there is a dictionary with the correspondence between
+the Visual Studio configuration (key, on the left) and a Conan profile (value, on the right).
+
+It is important tto take into account:
```
Comment (SSE4) [inline docs/README.md]:
```suggestion
It is important to take into account:
```

**Comment by jgsogo** [comment]
I've change the `Changelog` section to `Acknowledgements` (I think it is good for now) and added a README into the examples directories so we can add there a brief explanation about each one.

**Pull request merged**